iceoryx_posh/test/moduletests/test_posh_runtime_single_process.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2021 by Robert Bosch GmbH, Apex.AI Inc. All rights reserved.
```

Comment:
Copyright statement needs to be corrected

Reply:
What should I correct actually ?

Reply:
Please remove ``Apex.AI Inc``.